react/src/reactTextMask.js

```diff
@@ -52,12 +37,12 @@ export const MaskedInput = React.createClass({
     return (
       <input
         {...props}
-        onInput={this.onChange}
+        onInput={event => this.onChange(event)}
```

Comment:
Sure I can add it to the constructor. I just don't see it as a big deal since arrow functions are already more performant in node than bound ones https://github.com/nodejs/node/pull/3622